### PR TITLE
refactoring the auth service

### PR DESCRIPTION
**Pull request by leelakrishnabharadwaj**
refactoring the authentication service with better coding standards

### CODE REVIEW

File: src/main/java/com/auth/utils/JwtUtils.java [hunk 1]
```diff
@@ -6,72 +6,70 @@
...
-					.signWith(SignatureAlgorithm.HS512, secret).compact();
-		}
-
-		//validate token
-		public Boolean validateToken(String token, UserDetails userDetails) {
-			final String username = extractUsername(token);
-			return (username.equals(userDetails.getUsername()) && !isTokenExpired(token));
-		}
-	
+	public static final long JWT_TOKEN_VALIDITY = 5 * 60 * 60;
```
Comment (SumanjaliDamarla) [inline src/main/java/com/auth/utils/JwtUtils.java:20]: We can have this in the configuration file and read it from there.

File: src/main/java/com/auth/utils/JwtUtils.java [hunk 2]
```diff
@@ -6,72 +6,70 @@
...
+
+	private String secret = "Bharadwaj";
```
Comment (SumanjaliDamarla) [inline src/main/java/com/auth/utils/JwtUtils.java:22]: We can read this as well from the configuration file

File: src/main/java/com/auth/utils/JwtUtils.java (hunk 1 above)
Comment (leelakrishnabharadwaj) [reply src/main/java/com/auth/utils/JwtUtils.java:20]: config file kante we have something called vaults ..daniloki will migrate

Comment (SumanjaliDamarla) [reply src/main/java/com/auth/utils/JwtUtils.java:20]: We will use the vaults while **deploying** the configuration files, reading from the vault for each config during an API call is expensive and not in scope for now. The repo must be having a configuration file, I'll add it soon :)

**Comment by leelakrishnabharadwaj** [comment]
Implemented the same in my-experiments branch. Hence, closing this PR

**Pull request closed without merging**